oemoflex/model_structure/component_attrs/electricity-demand.csv

```diff
@@ -0,0 +1,8 @@
+attribute,type,unit,default,suffix,description
+region,str,n/a,n/a,,Country code
+name,str,n/a,,-electricity-demand,Name
+type,str,n/a,load,,Type
+carrier,str,n/a,electricity,,Energy carrier
+bus,str,n/a,,-electricity,The bus this component is connected to
+amount,float,GWh/a,n/a,,Total load over one year
+profile,str,n/a,n/a,-electricity-load-profile,Name of the profile
```

Comment:
Here I wonder why the names for the elements and their profiles use `demand` or `load` respectively.

Reply:
Unified by renaming 065f103